packages/core/src/App/Containers/AccountSwitcher/account-switcher-account-list.jsx

```diff
@@ -11,11 +11,13 @@ const AccountList = ({
     currency_icon,
     display_type,
     has_balance,
+    has_reset_balance,
     is_eu,
     is_disabled,
     is_virtual,
     loginid,
     onClickAccount,
+    onClickResetBalance,
```

Comment:
Is this function just for `virtual` accounts? If yes how about naming it `onClickVirtualResetBalance`?

Reply:
Alright, sure 👍